godaddy-email-marketing.php

```diff
@@ -151,6 +158,7 @@ public function init() {
 
 		// Initialize settings.
 		if ( is_admin() ) {
+			$this->display_gem_notice = get_option( 'wpem_gem_notice' );
```

Comment:
@EvanHerman Seems like we should be able to look for this inside the `action_admin_notices` method and don't need this private property at all. Or did you already try that?